### PR TITLE
Traveling mode et al

### PR DESCRIPTION
**Pull request by hemangandhi**
This is a smorgasbord of changes:
- Travelling data is verified on the server side (untested).
- The `validate` endpoint is tested. (Coverage is at 20%.)
- The deploy zip outputted includes the stage and files mentioned.
- Any mention of MLH login is dead (see [release 1.0](https://github.com/HackRU/lcs/releases/tag/1.0)).

### CODE REVIEW

**Comment by hemangandhi** [comment]
Ooh, this also resolves #40

File: authorize.py [hunk 1]
```diff
@@ -56,76 +47,14 @@ def authorize(event,context, is_mlh = False):
...
-    tests.update({"email":event['email']},{"$push":update_val})
+    tests.update_one({"email":event['email']},{"$push":update_val})
 
     #return the value pushed, that is, auth token with expiry time.
     #throw in the email for frontend.
     update_val['auth']['email'] = email
     ret_val = { "statusCode":200,"isBase64Encoded": False, "headers": { "Content-Type":"application/json" },"body" : json.dumps(update_val)}
     return config.add_cors_headers(ret_val)
 
-def mlh_callback(event, context):
```
Comment (Sail338) [inline authorize.py]: :'( rip mlh_callback hax

File: authorize.py [hunk 2]
```diff
@@ -136,16 +65,12 @@ def create_user(event, context, mlh = False):
...
-    if mlh == True:
-        event['password'] = "defacto"
-
     if 'password' not in event:
        return config.add_cors_headers({"statusCode":400, "body":"No password provided"})
 
     u_email = event['email']
     password = event['password']
-    password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt())
+    password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt(rounds=8))
```
Comment (Sail338) [inline authorize.py]: why 8? might wanna put this in config so we can change rounds when needed

Comment (hemangandhi) [reply authorize.py]: Tru, tho we'd have to invalidate people then, I think

**Pull request merged**